### PR TITLE
Code cleanup: Add space after closing brace where needed

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
## PR Summary

Using this [script](https://gist.github.com/SteveL-MSFT/f4e977caa3c44be2ce35f8012b151b4c) each folder is a separate commit making it easier to review.

The regular expression looks for any closing brace not followed by:

| catch, else, finally, another closing brace, newline, space, hash, slash (comment), * (comment)

In addition, because .NET RegEx doesn't support look behind, I skip any closing braces that are part of: { get; set; }

Some of the Copyright notices show changed because git resaved the file without BOM.

Manually reviewed all the changes.

Related https://github.com/PowerShell/PowerShell/pull/8500

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

**Comment by iSazonov** [comment]
@SteveL-MSFT Please resolve the merge conflit.

**Comment by SteveL-MSFT** [comment]
Let's try to get this reviewed and merge before merging other PRs otherwise it's going to be a merge conflict target

**Comment by iSazonov** [comment]
@SteveL-MSFT I think we need fix only my last comments about "while". Rest is minor.

File: src/Microsoft.WSMan.Management/ConfigProvider.cs [hunk 1]
```diff
@@ -2466,33 +2570,39 @@ private string EscapeValuesForXML(string value)
...
                             break;
                         }
+
                     case '<':
                         {
                             esc_str.Append("&lt;");
                             break;
                         }
+
                     case '>':
```
[the remark this answers is not in the document]
Comment (SteveL-MSFT) [reply src/Microsoft.WSMan.Management/ConfigProvider.cs]: As best I can tell, CodeFactor [doesn't complain](https://www.codefactor.io/repository/github/powershell/powershell/file/master/src/Microsoft.WSMan.Management/ConfigProvider.cs?category=Style&groupId=81&page=1) for `case` statements so I can add that to the ignore list and rerun

File: src/System.Management.Automation/engine/InitialSessionState.cs [hunk 2]
```diff
@@ -4019,6 +4062,7 @@ internal void ImportCmdletsFromAssembly(Assembly assembly, PSModuleInfo module)
             <#options#>          $options)
     }
 }
+
         ";
```
[the remark this answers is not in the document]
Comment (SteveL-MSFT) [reply src/System.Management.Automation/engine/InitialSessionState.cs]: Should also ignore if followed by a double quote

File: src/System.Management.Automation/cimSupport/cmdletization/ScriptWriter.cs [hunk 3]
```diff
@@ -1755,8 +1786,11 @@ private static BehaviorOnNoMatch GetBehaviorWhenNoMatchesFound(CmdletParameterMe
 function {0}
 {{
     {1}
+
```
[the remark this answers is not in the document]
Comment (SteveL-MSFT) [reply src/System.Management.Automation/cimSupport/cmdletization/ScriptWriter.cs]: Probably ok to skip if followed by opening brace

File: src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs [hunk 4]
```diff
@@ -2901,6 +2965,7 @@ protected override void ProcessRecord()
                                                    arguments,
                                                    csNotFoundMessageFormat,
                                                    _force}
+
```
[the remark this answers is not in the document]
Comment (SteveL-MSFT) [reply src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs]: Seems ok to skip if followed by closing parens

File: src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs [hunk 5]
```diff
@@ -1908,6 +1928,7 @@ internal string ReadLineWithTabCompletion(Executor exec)
                 }
 #endif
             }
+
             while (true);
```
[the remark this answers is not in the document]
Comment (SteveL-MSFT) [reply src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs]: Will skip if followed by `while`

**Comment by SteveL-MSFT** [comment]
@iSazonov thanks for taking the time to review this in detail.  I'll update the rules in my script and will rerun.

**Review by iSazonov — APPROVED** [review]

**Comment by iSazonov** [comment]
It was not easy, but I believe that it will facilitate our work in the future.

**Comment by iSazonov** [comment]
@SteveL-MSFT I'd merge if @daxian-dbw  is absent.

**Pull request merged**